redskyctl/internal/commands/authorize_cluster/generator.go

```diff
@@ -67,7 +66,7 @@ func NewGeneratorCommand(o *GeneratorOptions) *cobra.Command {
 		Long:  "Generate authorization secret for Red Sky Ops",
 
 		Annotations: map[string]string{
-			commander.PrinterAllowedFormats: "json,yaml",
+			commander.PrinterAllowedFormats: "json,yaml,helm",
```

Comment:
it'd be nice if we could generate these from what's defined through `commander.SetKubePrinter` to avoid mismatches. not something for now though.